javascript/src/bigint.ts

```diff
@@ -32,3 +32,27 @@ export function uint8ArrayToBigInt(v: Uint8Array): bigint {
     }
     return BigInt(hexStr);
 }
+
+export function uint64ToUint8Array(v: bigint): Uint8Array {
+    let hex = v.toString(16);
+    // if (16 < hex.length) {
```

Comment:
please uncomment. its good that if someone tries to translate a 65 bit (hex will be bigger than 16) number it fails rather then save only the first 64 bits.